### DISCUSSION TITLE
Any reason why git branch information not being retrieved in pkg/context/contex.go

### DISCUSSION THREAD

**Discussion by obourdon** [goreleaser/goreleaser#1635] [Q&A] [answered]
Even though I did spend some time (but may be not enough) in the goreleaser documentation, I am wondering why the branch information (like in the output of `git rev-parse --abbrev-ref HEAD`) is not retrieved and therefore can not be used like all others (Full/Short)Commit, Version. Thanks in advance

**Comment by caarlos0** [comment]
Mostly because the idea is to run against a tag, in which case the branch is not very relevant I think...

what's your use case for that?

**Comment by obourdon** [comment, marked as answer]
@caarlos0 many thanks for answering and sorry for not replying back earlier due to vacations ;-)

Sometimes, we produce feature branches that can also be tagged independently of master branch and we find it way easier to have BRANCHNAME-SMALLCOMMITID (like in docker images) rather than only a tag. I agree that this could seem very personal preference but I definitely think this should not be very difficult to add ... (will try to have a look if I have some spare time but not sure this will be the case in the short term)
